artifactory/cli.go

```diff
@@ -854,15 +873,20 @@ func getDockerFlags() []cli.Flag {
 	return flags
 }
 
-func getNpmCommonFlags() []cli.Flag {
+//These flags are not valid for native npm command
+func getNpmLegacyFlages() []cli.Flag {
 	npmFlags := []cli.Flag{
 		cli.StringFlag{
 			Name:  "npm-args",
 			Usage: "[Optional] A list of npm arguments and options in the form of \"--arg1=value1 --arg2=value2\"` `",
```

Comment:
I suggest adding **[Deprecated]** to all deprecated options (both npm and go) as follows:
Usage: "[Optional] [Deprecated] A list of npm arguments and options in the form of \"--arg1=value1 --arg2=value2\"` `",